web/doppelkopf.html

```diff
@@ -3,15 +3,28 @@
     <title>
       Scorepads - Doppelkopf
     </title>
+    <link rel="stylesheet" href = "./css/doppelkopf.css"></link>
     <script type="text/javascript" src="./js/doppelkopf.js"></script>
   </head>
   <body>
     <h1>
       Doppelkopf
     </h1>
-    <div>
-      <select id="player">
-      </select>
+    <div class = "mainArea">
+      <p>Wähle die teilnehmenden Spieler aus!</p>
+      <fieldset>
+          <select id="player1">
+            </select>
+            <select id="player2">
+            </select>
+            <select id="player3">
+            </select>
+            <select id="player4">
+            </select>
+      </fieldset>
+      <input type="button" value="Spiel beginnen" onclick="window.location.replace('doppelkopfSchreiben.html')"/>
```

Comment:
Wenn du einfach weiterleiten willst, reicht `onclick="window.location.href = 'doppelkopfSchreiben.html'"`. Mit `window.location.replace` ersetzt du den Eintrag in der Browser History, dann funktioniert die `Zurück`-Funktion im Browser nicht mehr :D

Reply:
Generell ist es immer eine Gute Idee solche Funktionen nicht in HTML zu definieren, sondern in JavaScript selbst. Dort kannst du einfach die onclick, onchange, onload oder was auch immer definieren, und bist dann viel flexibler was die Ausführung und Definition angeht. Bei den `<select>` Elementen oben hast du es schon so gemacht, das würde ich überall so durchziehen.

Ähnlich ist es auch bei dem `onload` in `<body>` in `doppelkopfSchreiben.html`, das würde ich auch in JavaScript definieren. Vermutlich macht es sogar Sinn, für `doppelkopfSchreiben.html` eine eigene JavaScript Datei anzulegen, denn momentan wird in beiden Doppelkopf-Dateien dasselbe gemacht, obwohl die zweite Seite ja gar keine `<select>` Elemente und dergleichen hat.